tests/utils.py

```diff
@@ -1,17 +1,19 @@
 from __future__ import annotations
 
-import contextlib
+__all__ = ["http_server"]
```

Comment:
```suggestion
__all__ = ("http_server",)
```

just a pet-peeve 😄 , but can we use tuples for `__all__`, after all they should be immutable no?

Reply:
_Should_ be, yep.. can you point to some documentation that says that `tuple` is OK as well as `list`?  (I couldn't find any, initially)

Reply:
I can't argue with your logic, but I've never seen a tuple in the wild

Reply:
mypy (https://github.com/python/mypy/pull/1640) and pylance recognise any sequence of strings

> I can't argue with your logic, but I've never seen a tuple in the wild

I always use them 😉  https://github.com/executablebooks/markdown-it-py/blob/1064835b30325e857252f5b67b34e622261723ca/markdown_it/__init__.py#L2

Reply:
I guess this is the "formal" documentation: https://github.com/python/typing/blob/main/docs/source/libraries.rst#library-interface-public-and-private-symbols

Reply:
For `__all__` I tend to use a list because I don't want to put the comma if I only have a single element. It's only by laziness that I'm doing this though.

Reply:
> I guess this is the "formal" documentation: https://github.com/python/typing/blob/main/docs/source/libraries.rst#library-interface-public-and-private-symbols

Thanks, perfect.  I've derived a permalinked from that, https://github.com/python/typing/blob/9a39406fa45fbfa76eb5d35afb7863f5f727ad87/docs/source/libraries.rst?plain=1#L108-L119 - and will use tuples and reference that.

> For `__all__` I tend to use a list because I don't want to put the comma if I only have a single element. It's only by laziness that I'm doing this though.

Agreed it looks a little unusual.  However the immutability benefit does seem genuine, so I'll apply this.